eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -79,13 +80,17 @@ def update_enrollment(*args, **kwargs):
             raise APIException('No user found with that email')
         else:
             username = match.username
+            email = match.email
     LOG.info('Updating enrollment for student: %s of course: %s mode: %s', username, course_id, mode)
     enrollment = api._data_api().update_course_enrollment(username, course_id, mode, is_active)
     if not enrollment:
         raise APIException('No enrollment found for {}'.format(username or email))
     if enrollment_attributes is not None:
         api.set_enrollment_attributes(username, course_id, enrollment_attributes)
 
+    enrollment['enrollment_attributes'] = enrollment_attributes
+    enrollment['course_id'] = course_id
+    enrollment['email'] = email
```

Comment:
I know I asked for it, but it was my bad, we should not put the email in here

Reply:
done.